src/Types/CustomShape.ts

```diff
@@ -0,0 +1,6 @@
+import type { SingleOrMultiple } from "./SingleOrMultiple";
+import type { IShapeValues } from "../Interfaces/Options/Particles/Shape/IShapeValues";
+
+export type CustomShape = {
+    [type: string]: SingleOrMultiple<IShapeValues>
+};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)